playbooks/roles/openvpn/templates/client-common.j2

```diff
@@ -7,6 +7,7 @@ nobind
 persist-key
 persist-tun
 ns-cert-type server
+tls-version-min 1.2
```

Comment:
I like the idea of this, but we'll have to make sure that it won't break existing clients. The official iOS OpenVPN client, for example, didn't work well when we tried to [update to a fancier set of cipher suites](https://github.com/jlund/streisand/commit/cb65bfc676e88b5ca9843c9dcfb8aa3aa3cb3359#diff-d667e36408c9400858277f3ff051f16b) last year.

Which clients did you test this with?